workers/social/lib/social/render/scriptblock.coffee

```diff
@@ -70,8 +69,8 @@ module.exports = (options = {}, callback) ->
       };
     </script>
 
-    <script src="/a/p/p/#{KONFIG.version}/bundle.vendor.js"></script>
-    <script src="/a/p/p/#{KONFIG.version}/bundle.main.js"></script>
+    <script src="/a/p/p/#{KONFIG._CLIENTVERSION}/bundle.vendor.js"></script>
+    <script src="/a/p/p/#{KONFIG._CLIENTVERSION}/bundle.main.js"></script>
```

Comment:
```
workers/social/lib/social/render/loggedout/kodinghome.coffee
workers/social/lib/social/render/loggedout/tags/hackathon.coffee
workers/social/lib/social/render/loggedout/tags/landing.coffee
```
these are not using main app styles/app files so we can safely ignore these, but
```
workers/social/lib/social/render/styleblock.coffee
```
this one important that I missed, thanks!